modules/cudl-data-processing/lambda.tf

```diff
@@ -209,9 +209,9 @@ resource "aws_lambda_event_source_mapping" "sqs-trigger-lambda-transforms" {
   }
 }
 
-resource "aws_lambda_event_source_mapping" "sqs-trigger-lambda-db" {
-  count = length(var.db-lambda-information)
-
-  event_source_arn = aws_sqs_queue.db-lambda-sqs-queue[count.index].arn
-  function_name    = aws_lambda_function.create-db-lambda-function[count.index].arn
-}
+# resource "aws_lambda_event_source_mapping" "sqs-trigger-lambda-db" {
+#   count = length(var.db-lambda-information)
+#
+#   event_source_arn = aws_sqs_queue.db-lambda-sqs-queue[count.index].arn
+#   function_name    = aws_lambda_function.create-db-lambda-function[count.index].arn
+# }
```

Comment:
This section could be removed as it will still exist in git history

Reply:
Removed